src/components/Book.js

```diff
@@ -1,13 +1,23 @@
+/*eslint-disable */
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/booksSlice';
+import { deleteBook, getBookItems } from '../redux/books/booksSlice';
 
-// genre, title, author,
 function Book({
-  id, title, author, category, percentage, chapter,
+  item_id, title, author, category, percentage, chapter,
 }) {
   const dispatch = useDispatch();
 
+  async function handleRemove(e) {
+    e.preventDefault();
+    try {
+      await dispatch(deleteBook(item_id));
+      await dispatch(getBookItems());
+    } catch (error) {
+      console.log(error)
```

Comment:
- [optional] please kindly remove the `console.log` from your code like the previous reviewer mentioned.kindly implement that for all components.